src/components/MoneyRequestHeader.js

```diff
@@ -0,0 +1,127 @@
+import styles from '../styles/styles';
+import themeColors from '../styles/themes/default';
+import CONST from '../CONST';
+import withWindowDimensions from './withWindowDimensions';
+import compose from '../libs/compose';
+import Navigation from '../libs/Navigation/Navigation';
+import ROUTES from '../ROUTES';
+
+const propTypes = {
+    /** The report currently being looked at */
+    report: iouReportPropTypes.isRequired,
+
+    /** The policies which the user has access to and which the report could be tied to */
+    policies: PropTypes.shape({
+        /** Name of the policy */
+        name: PropTypes.string,
+    }).isRequired,
+
+    personalDetails: PropTypes.objectOf(participantPropTypes).isRequired,
+
+    isSingleTransactionView: PropTypes.bool,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    isSingleTransactionView: false,
+};
+
+const MoneyRequestHeader = (props) => {
+    const formattedAmount = props.numberFormat(props.report.total / 100, {
+        style: 'currency',
+        currency: props.report.currency,
+    });
+    const isSettled = /* ReportUtils.isSettled(props.report.reportID); */ false;
+    const workspaceName = ReportUtils.getPolicyName(props.report, props.policies);
+    const workspaceAvatar = ReportUtils.getWorkspaceAvatar(props.report, props.policies);
+    return (
+        <View style={[
+            {backgroundColor: themeColors.highlightBG},
+            styles.pl0,
+        ]}
+        >
+            <HeaderWithCloseButton
+                shouldShowAvatarWithDisplay
+                shouldShowThreeDotsButton={!isSettled}
+                threeDotsMenuItems={[{
+                    icon: Expensicons.Trashcan,
+                    text: props.translate('common.delete'),
+                    onSelected: () => {},
+                }]}
+                threeDotsAnchorPosition={styles.threeDotsPopoverOffsetNoCloseButton}
+                report={props.report}
+                policies={props.policies}
+                personalDetails={props.personalDetails}
+                containerStyles={[styles.pt5, styles.pb3]}
+                shouldShowCloseButton={false}
+                shouldShowBackButton={props.isSmallScreenWidth}
+                onBackButtonPress={() => Navigation.navigate(ROUTES.HOME)}
+            />
+            <View style={[styles.ph5, styles.pb5]}>
+                <Text style={[styles.textLabelSupporting, styles.lh16]}>To</Text>
+                <View style={[
+                    styles.flex1,
+                    styles.flexRow,
+                    styles.alignItemsCenter,
+                    styles.justifyContentBetween,
+                    styles.pv3,
+                ]}
+                >
+                    <View style={[
+                        styles.flexRow,
+                        styles.alignItemsCenter,
+                        styles.justifyContentBetween,
+                    ]}
+                    >
+                        <Avatar
```

Comment:
In the IOU case, we should display the avatar and details of the payee here.

Reply:
In the interest of unblocking other issues, maybe we can split that into a separate issue?

Reply:
I think this is resolved